sample01.go

```diff
@@ -1,7 +1,9 @@
 package sample01
 
 func HelloWorld(s string) string {
-	var x int
-	x = "string"
+	var y int
+	y = "aaa" // TODO
 	return "hello world, " + s
 }
+
+var ExportedVarWithoutComment = 1 // TODO: revert
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported var ExportedVarWithoutComment should have comment or be unexported